crates/fluvio-version-manager/src/command/itself/install.rs

```diff
@@ -0,0 +1,102 @@
+use std::env::current_exe;
+use std::fs::{create_dir, copy, write};
+use std::path::PathBuf;
+
+use clap::Parser;
+use color_eyre::eyre::Result;
```

Comment:
use anyhow

Reply:
Oh! I moved it to `color_eyre` given that it provides good UserEx for error reporting.
Now that `fvm` CLI only holds presentational logic, errors bubble up to `main`, the `color_eyre` `Error` includes Error reporting features.

Here is an example from docs:

```
color-eyre on  hooked [$!] is 📦 v0.5.0 via 🦀 v1.44.0
❯ cargo run --example custom_section
    Finished dev [unoptimized + debuginfo] target(s) in 0.04s
     Running `target/debug/examples/custom_section`
Error:
   0: Unable to read config
   1: cmd exited with non-zero status code

Stderr:
   cat: fake_file: No such file or directory

  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ SPANTRACE ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

   0: custom_section::output2 with self="cat" "fake_file"
      at examples/custom_section.rs:14
   1: custom_section::read_file with path="fake_file"
      at examples/custom_section.rs:58
   2: custom_section::read_config
      at examples/custom_section.rs:63

Suggestion: try using a file that exists next time
```

Source: https://docs.rs/color-eyre/latest/color_eyre/#tldr

Reply:
This kind of stack trace is only relevant for developers.   This has been removed from fluvio CLI.    Formatting of error should be decoupled from error handling itself.